collections/src/main/scala/strawman/collection/Seq.scala

```diff
@@ -82,34 +82,14 @@ object Seq extends SeqFactory.Delegate[Seq](immutable.Seq)
   * @define Coll `Seq`
   */
 trait SeqOps[+A, +CC[_], +C] extends Any
-  with IterableOps[A, CC, C] {
+  with IterableOps[A, CC, C]
+  with ArrayLike[A] {
```

Comment:
I’m not sure it’s a good idea to have `Seq` extend `ArrayLike`.

`ArrayLike` suggests that the size can be known in O(1), but that’s not the case for all `Seq`s (e.g. `LinearSeq`).